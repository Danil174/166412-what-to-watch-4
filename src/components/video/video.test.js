import React from "react";
import renderer from "react-test-renderer";
import Video from "./video.jsx";

describe(`TestVideoComponent`, () => {
  it(`render Video component`, () => {
    const tree = renderer
      .create(
          <Video>
            <video></video>
          </Video>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
