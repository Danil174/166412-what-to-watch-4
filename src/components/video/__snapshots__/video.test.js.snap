// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestVideoComponent render Video component 1`] = `
<video
  height={175}
  width={280}
/>
`;
